queries.sql

```diff
@@ -23,3 +24,64 @@ SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
 /* Find all animals with a weight between 10.4kg and 17.3kg */
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+--DAY 02
+/* Inside a transaction update the animals table by setting the species column to unspecified. 
+Verify that change was made. 
+Then roll back the change and verify that the species columns went back to the state before the transaction. */
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+/*Check changes*/
+SELECT * FROM animals; 
+ROLLBACK;
+/*Check that table is restored to the state before the transaction*/
+SELECT * FROM animals; 
+
+/*Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.*/
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+/*Update the animals table by setting the species column to pokemon for all animals that don't have species already set.*/
+UPDATE animals
+SET species = 'pokemon'
+WHERE name NOT LIKE '%mon';
```

Comment:
- [ ] Dear @JuanDiegoGuerra, am sure this query interprets the requirement, you are just updating any species to Pokemon where the name is anything not similar to mon, but the requirement says to update the animals table by setting the species column to Pokemon for all animals that `don't have species already set` I think it refers to a `null` value 
<img width="957" alt="1" src="https://github.com/JuanDiegoGuerra/Vet-Clinic_Database/assets/77843050/d2510ed1-d541-4b18-ad79-f47ca290b9bd">